sentry/templates/deployment-sentry-cron.yaml

```diff
@@ -61,7 +61,15 @@ spec:
           - name: C_FORCE_ROOT
             value: "true"
           - name: BROKER_URL
+            {{- if .Values.rabbitmq.enabled }}
             value: amqp://{{ .Values.rabbitmq.username }}:{{ .Values.rabbitmq.password }}@{{ .Values.rabbitmq.host }}:5672
+            {{- else if .Values.redis.enabled }}
```

Comment:
`{{- else }}` is enough here. One of those is required anyways

Reply:
fixed